src/initializers/diamorphosis.ts

```diff
@@ -68,14 +68,18 @@ export default (config, orkaOptions: Partial<OrkaOptions>) => {
     enabled: true,
     inbound: {
       request: {
-        enabled: false
-      }
+        enabled: false,
+        ...config.riviere?.inbound?.request
+      },
+      ...config.riviere?.inbound
```

Comment:
❓ @klesgidis in order to not overwrite request.enabled:false this should be on the top right?